lib/cucumber/core/test/result.rb

```diff
@@ -5,13 +5,19 @@ module Core
     module Test
       module Result
 
+        # Defines a status reader on a result class for the given status
+        #
         # Defines predicate methods on a result class with only the given one
         # returning true
-        def self.status_queries(status)
+        def self.status_methods(status)
           Module.new do
+            define_method :status do
```

Comment:
Not picky at all, the status _is_ the result, it's reflected in the class name and the predicate methods so it makes more sense to expose it as a _version_ of the result (i.e. it's symbol form) rather than a separate piece of data.